src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.html

```diff
@@ -38,6 +38,37 @@ <h3>{{ 'artemisApp.pages.tutorialGroupDetail.title' | artemisTranslate }}:</h3>
                 <span>{{ 'artemisApp.pages.tutorialGroupDetail.teachingAssistantNotSet' | artemisTranslate }}</span>
             </ng-template>
         </dd>
+        <!-- Utilization -->
+        <dt>
+            <span>{{ 'artemisApp.entities.tutorialGroup.utilization' | artemisTranslate }}</span>
+            <fa-icon [icon]="faQuestionCircle" class="text-secondary" ngbTooltip="{{ 'artemisApp.entities.tutorialGroup.utilizationHelp' | artemisTranslate }}"></fa-icon>
+        </dt>
+        <dd>
+            <ng-container *ngIf="tutorialGroup.averageAttendance !== undefined && tutorialGroup.capacity !== undefined; else noUtilization">
+                <jhi-vertical-progress-bar
+                    [fillLevelInPercent]="Math.round((tutorialGroup.averageAttendance / tutorialGroup.capacity) * 100)"
+                    [widthInPixels]="20"
+                    [heightInPixels]="50"
+                >
+                </jhi-vertical-progress-bar>
+            </ng-container>
+            <ng-template #noUtilization>
+                <span>{{ '' }}</span>
+            </ng-template>
+        </dd>
+        <!-- Average Attendance -->
+        <dt>
+            <span>{{ 'artemisApp.entities.tutorialGroup.averageAttendance' | artemisTranslate }}</span>
+            <fa-icon [icon]="faQuestionCircle" class="text-secondary" ngbTooltip="{{ 'artemisApp.entities.tutorialGroup.averageAttendanceHelp' | artemisTranslate }}"></fa-icon>
+        </dt>
+        <dd>
+            <ng-container *ngIf="tutorialGroup.averageAttendance !== undefined; else noAverageAttendance">
+                <span>{{ Math.round(tutorialGroup.averageAttendance) }}</span>
```

Comment:
Same here, so we don't need to create `Math` reference.

Reply:
Personally I think this is really a matter of taste tbh.